tools/accuracy_checker/accuracy_checker/evaluators/quantization_model_evaluator.py

```diff
@@ -143,15 +134,16 @@ def _set_number_infer_requests(nreq):
                         ir
                     ) = ready_data
                     batch_predictions = _process_ready_predictions(
-                        batch_raw_predictions, batch_identifiers, batch_meta, self.adapter
+                        batch_raw_predictions, batch_identifiers, batch_meta
                     )
                     free_irs.append(ir)
                     annotations, predictions = self.postprocessor.process_batch(
                         batch_annotation, batch_predictions, batch_meta
                     )
```

Comment:
I'm trying to run a classification model and `process_batch` returns a tuple of empty lists because of empty `batch_annotation`

Reply:
ok, I will fix it

Reply:
@evgeniya-egupova, please try again